bin/integration_test.sh

```diff
@@ -22,7 +30,7 @@ reportfail() {
     echo ""
 }
 
-trap reportfail 0
+trap "reportfail ." 0
```

Comment:
Funny story--because `reportfail` looks at `$1`, if the script failed, this would cause it to fail on the trap because it wasn't passing in an argument.